src/libs/Pusher/index.ts

```diff
@@ -21,7 +21,8 @@ import type {
 import type PusherModule from './types';
 
 let shouldForceOffline = false;
-Onyx.connect({
+// We have used `connectWithoutView` here because it is not connected to any UI
```

Comment:
Would you mind adding a little more context to this comment, like:

> shouldForceOffline is only used to ignore pusher events when the client has been forced offline. Since it's not connected to any UI, it's OK to use connectWithoutView